src/openms/thirdparty/IsoSpec/IsoSpec/marginalTrek++.cpp

```diff
@@ -204,8 +204,8 @@ smallest_lprob(atomCnt * *std::min_element(atom_lProbs, atom_lProbs+isotopeNo))
 {
     try
     {
-        if(ISOSPEC_G_FACT_TABLE_SIZE-1 <= atomCnt)
-            throw std::length_error("Subisotopologue too large, size limit (that is, the maximum number of atoms of a single element in a molecule) is: " + std::to_string(ISOSPEC_G_FACT_TABLE_SIZE-1));
+        // if(ISOSPEC_G_FACT_TABLE_SIZE-1 <= atomCnt)
+        //     throw std::length_error("Subisotopologue too large, size limit (that is, the maximum number of atoms of a single element in a molecule) is: " + std::to_string(ISOSPEC_G_FACT_TABLE_SIZE-1));
```

Comment:
instead of failing when a value wont fit in the table, calculate value on the fly instead of using lookup

```
if (n >= ISOSPEC_G_FACT_TABLE_SIZE) return -lgamma(n+1);
```